pygdf/series.py

```diff
@@ -688,35 +694,35 @@ def find_last_value(self, value):
     #
     # Stats
     #
-    def count(self):
+    def count(self, axis=None, skipna=True):
         """The number of non-null values"""
         return self.valid_count
 
-    def min(self):
+    def min(self, axis=None, skipna=True):
         """Compute the min of the series
         """
         return self._column.min()
 
-    def max(self):
+    def max(self, axis=None, skipna=True):
         """Compute the max of the series
         """
         return self._column.max()
 
-    def sum(self):
+    def sum(self, axis=None, skipna=True):
         """Compute the sum of the series"""
         return self._column.sum()
 
-    def mean(self):
+    def mean(self, axis=None, skipna=True):
         """Compute the mean of the series
         """
         return self._column.mean()
 
-    def std(self, ddof=1):
+    def std(self, ddof=1, axis=None, skipna=True):
         """Compute the standard deviation of the series
         """
         return np.sqrt(self.var(ddof=ddof))
 
-    def var(self, ddof=1):
+    def var(self, ddof=1, axis=None, skipna=True):
```

Comment:
I probably need to add input validation to ensure that the user doesn't specify anything *except* for the defaults.   This silently produces the wrong result if they ask for something other than the defaults.

Reply:
yes, that would be nice. Other parts of the *gdf ecosystem use simple asserts to cover the unsupported APIs. eg

```py
assert axis is None and skipna
```